send-transaction-service/src/send_transaction_service.rs

```diff
@@ -777,6 +783,26 @@ impl SendTransactionService {
             .unwrap_or_else(|| vec![tpu_address])
     }
 
+    fn get_tpu_addresses_with_slots<'a, T: TpuInfo>(
+        tpu_address: &'a SocketAddr,
+        leader_info: Option<&'a T>,
+        config: &'a Config,
+        protocol: Protocol,
+    ) -> Vec<(&'a SocketAddr, Slot)> {
+        let addresses = leader_info.as_ref().map(|leader_info| {
+            leader_info.get_leader_tpus_with_slots(config.leader_forward_count, protocol)
+        });
+        addresses
+            .map(|address_list| {
```

Comment:
not sure what's intended here, can't call `map()` directly on a `Vec`

Reply:
Note the map at line 792, addresses is of type Option<Vec<(_, _)>>

Reply:
ah, damned `fmt`...